### ISSUE TITLE
[Button]: Styles for button do not change to Active while JAWS tool is launched

### ISSUE THREAD

**Issue by zagmax** [SAP/ui5-webcomponents#11344]
### Bug Description

When button is pressed via click or keyboard it displays the Active state, but when JAWS a11y tool is launched the button does not display the style change on button press.

JAWS 2024.2411.16

https://github.com/user-attachments/assets/a585ab1a-14e8-437a-b927-72715c149b3f

Note: styles are still applied when button is clicked by mouse.

### Affected Component

Button

### Expected Behaviour

Styles always apply

![Image](https://github.com/user-attachments/assets/10797304-bfe6-4531-9ef1-a0dbe7d02da3)

### Steps to Reproduce

1. Open playground for either version (1.24.17 \ 2.9.0)
2. Launch JAWS tool
3. Reach button via keyboard controls
4. Press Enter\Space
5. Pay attention to style of the button
...

### Priority

None

### UI5 Web Components Version

 1.24.17, 2.9.0

### Browser

Chrome

### Operating System

Windown 10 \ 11

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Valkanister** [comment]
Hello @zagmax , 

The on-screen keyboard is a feature of JAWS and we cannot control how it interacts with the components. A physical keyboard moves the focus and sets the state correctly. 

Kind regards, 

Georgi

SAP Development Support

**Comment by zagmax** [comment]
Hi Georgi, 

The on-screen keyboard in the recording is not a JAWS feature but from Windows itself, and was put there only to have a visual display what is being pressed and what result it brings. 

While using the regular keyboard the actual effect is the same - styles are not applied. 
Just to clarify, issue is not related to focus, but to the active style display (in this example the additional border around the button)

**Comment by Valkanister** [comment]
Hello @SAP/ui5-webcomponents-topic-b 
The button is not triggering its Active state when triggered by an on-screen keyboard. 
Please check if this is a valid interaction scenario. 

Kind regards, 

Georgi

SAP Development Support

**Comment by unazko** [comment]
Hello @zagmax,

The described is a JAWS related issue reproducible with native HTML as well: https://jsfiddle.net/hs9efz5u/
There is a logged issue to FreedomScientific: https://github.com/FreedomScientific/standards-support/issues/899
and to NVDA about the same: https://github.com/nvaccess/nvda/issues/4903
Therefore I'm closing the incident as there is nothing we can do at our side.

Best regards,
Boyan
